pyproject.toml

```diff
@@ -52,74 +50,74 @@ verbose = 0
 [tool.ruff]
 line-length = 80
 lint.select = ["ALL"]
-exclude=[]
-lint.ignore = [  # NOTE: non-permanent exclusions should be added to `.ruff.toml` instead.
+exclude = []
+lint.ignore = [ # NOTE: non-permanent exclusions should be added to `.ruff.toml` instead.
 
-    # flake8-builtins (A) : shadowing a Python built-in.
-    # New ones should be avoided and is up to maintainers to enforce.
-    "A00",
+  # flake8-builtins (A) : shadowing a Python built-in.
+  # New ones should be avoided and is up to maintainers to enforce.
+  "A00",
 
-    # flake8-annotations (ANN)
-    "ANN101",  # No annotation for `self`.
-    "ANN102",  # No annotation for `cls`.
+  # flake8-annotations (ANN)
+  "ANN101", # No annotation for `self`.
+  "ANN102", # No annotation for `cls`.
 
-    # flake8-bugbear (B)
-    "B008",  # FunctionCallArgumentDefault
+  # flake8-bugbear (B)
+  "B008", # FunctionCallArgumentDefault
 
-    # flake8-commas (COM)
-    "COM812",  # TrailingCommaMissing
-    "COM819",  # TrailingCommaProhibited
+  # flake8-commas (COM)
+  "COM812", # TrailingCommaMissing
+  "COM819", # TrailingCommaProhibited
 
-    # pydocstyle (D)
-    # Missing Docstrings
-    "D102",  # Missing docstring in public method. Don't check b/c docstring inheritance.
-    "D105",  # Missing docstring in magic method. Don't check b/c class docstring.
-    # Whitespace Issues
-    "D200",  # FitsOnOneLine
-    # Docstring Content Issues
-    "D410",  # BlankLineAfterSection. Using D412 instead.
-    "D400",  # EndsInPeriod.  NOTE: might want to revisit this.
+  # pydocstyle (D)
+  # Missing Docstrings
+  "D102", # Missing docstring in public method. Don't check b/c docstring inheritance.
```

Comment:
I'd suggest not formatting these files since this pull request is not about formatting. This confuses the reviewer on what is actually changed.

Reply:
This was automatic, I will try to remove it.